cad/LibreCAD/Portfile

```diff
@@ -2,10 +2,44 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
-PortGroup           qmake 1.0
+PortGroup           qmake5 1.0
+PortGroup           cxx11 1.1
+
+name                LibreCAD
+subport             LibreCAD-devel {}
+
+qt5.depends_component qtsvg qttranslations qttools
+
+if {${subport} eq "LibreCAD"} {
+
+    conflicts           LibreCAD-devel
+    github.setup        LibreCAD LibreCAD 2.1.3
+    checksums           rmd160  2f50b879e95c0ecc019dd5b7f6a218f62744cc1b \
+                        sha256  5d0bc37efa1006742785c06ec30737c7866b0366048553c5560e06a3e442f7b3 \
+                        size    22416015
+                 
+    post-extract {
+        reinplace "s|lrelease|${qt_bins_dir}/lrelease|g" scripts/postprocess-osx.sh
+    }
+
+} elseif {${subport} eq "LibreCAD-devel"} {
```

Comment:
Same with `elseif  {${subport} eq "LibreCAD-devel"}` which can be written as `subport ${name}-devel`